Mathlib/Algebra/Group/Defs.lean

```diff
@@ -97,14 +97,46 @@ section Mul
 
 variable [Mul G]
 
-/-- `left_mul g` denotes left multiplication by `g` -/
+/-- `leftMul g` denotes left multiplication by `g` -/
 @[to_additive "`left_add g` denotes left addition by `g`"]
 def leftMul : G → G → G := fun g : G => fun x : G => g * x
 
-/-- `right_mul g` denotes right multiplication by `g` -/
+/-- `rightMul g` denotes right multiplication by `g` -/
 @[to_additive "`right_add g` denotes right addition by `g`"]
 def rightMul : G → G → G := fun g : G => fun x : G => x * g
 
+/-- A mixin for left cancellative multiplication. -/
+class IsLeftCancelMul (G : Type u) [Mul G] : Prop where
```

Comment:
```suggestion
class LeftCancelMul (G : Type u) [Mul G] : Prop where
```
Isn't that the correct name?

Reply:
I am not sure, we have [IsEmpty](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Logic/IsEmpty.html#IsEmpty).

Reply:
Actually, I think this is the whole point: `LeftCancelMul` would mean writing `G [LeftCancelMul G]` also gives the multiplication, that is exactly what I *don't* want.

Reply:
IsLeftCancelMul is correct.